### ISSUE TITLE
Does Rubix ML support Online learning?

### ISSUE THREAD

**Issue by flavio-schoute** [RubixML/ML#216]
Hi,

I have question, is it possible to make the model dynamic. So basicly trained online. That is, data is continually entering the system and we're incorporating that data into the model through continuous updates. Is this possible with RubixML?

**Comment by andrewdalpino** [comment]
Yes @Snicser, Online Learners are quite common in Rubix ML. I suggest reading the docs for the details.

https://docs.rubixml.com/1.0/training.html#batch-vs-online-learning

https://docs.rubixml.com/1.0/online.html

A list of the Online Learners can be found here:

https://docs.rubixml.com/1.0/choosing-an-estimator.html
